src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import static edu.wpi.first.units.Units.Rotations;
+
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ClimberConstants;
+import frc.robot.Constants.MotorIDConstants;
+
+public class Climber extends SubsystemBase {
+  /** Creates a new Climber. */
+  private TalonFX climberMotorLeader;
+
+  private TalonFX climberMotorFollower;
+  private PIDController climberPID;
+
+  public Climber() {
+    climberMotorLeader = new TalonFX(MotorIDConstants.kCLimberMotorLeader);
+    climberMotorFollower = new TalonFX(MotorIDConstants.kCLimberMotorFollower);
+    climberMotorFollower.setControl(new Follower(MotorIDConstants.kCLimberMotorLeader, true));
```

Comment:
I'm not sure this should be setup as a follower. The two climb pieces are not mechanically linked, are they?